glue_job_create_csv.tf

```
@@ -62,6 +62,9 @@ resource "aws_glue_job" "create_csv" {
     "--subscriber"     = var.csv_jobs[count.index]["Subscriber"]
     "--receiver"       = var.csv_jobs[count.index]["Receiver"]
     "--verbose"        = "true"
+    // Updates to all-fields version of job
+    "--deleteMetadataFile" = "true"   // Optional, but recommended
+    "--workgroup"      = "primary"    // May require adjustment depending on workgroup settings
```

The workgroup is a terraform module you can access by using:


```suggestion
    "--workgroup"      = aws_athena_workgroup.shepherd[count.index]
```

You can have the script default to `primary` but the ETL job doesn't have access to talk to that workgroup.